app/src/main/java/org/mozilla/focus/session/ui/SessionViewHolder.java

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.session.ui;
+
+import android.animation.Animator;
+import android.animation.AnimatorListenerAdapter;
+import android.graphics.Color;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.graphics.drawable.DrawableCompat;
+import android.support.v7.content.res.AppCompatResources;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+
+import org.mozilla.focus.R;
+import org.mozilla.focus.session.Session;
+import org.mozilla.focus.session.SessionManager;
+import org.mozilla.focus.utils.UrlUtils;
+
+import java.lang.ref.WeakReference;
+
+public class SessionViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
```

Comment:
\[PMD\] INFO: The class 'SessionViewHolder' has a Cyclomatic Complexity of 3 \(Highest = 5\)\.